pkg/kube/secrets.go

```diff
@@ -80,6 +81,7 @@ func NewSecret(cred types.Credential, owner *metav1.OwnerReference, namespace st
 	}
 
 	secret = PrepareNewSecret(cred, caSecret, connectJsonHost)
+	secret.ObjectMeta.Labels = cred.Labels
```

Comment:
In case of secrets generated by the service controller (enabling TLS with generated credentials) this field will end up empty (https://github.com/skupperproject/skupper/blob/06f42e106da2a307f4853925030286b432e77edc/pkg/kube/qdr/tls_handler.go#L147).

One option would be to get the labels of the CA if there are none specified for the credentials, or to make  `tls_handler.go` functions responsible of that.

Reply:
I've added a commit that adds the site labels to generate service tls secrets.